src/shared/Git-Credential-Manager/Program.cs

```diff
@@ -75,7 +74,7 @@ public static void Main(string[] args)
                                   .ConfigureAwait(false)
                                   .GetAwaiter()
                                   .GetResult();
-
+                
```

Comment:
Do you have a setting in your IDE to trim trailing whitespace on save? Could help to avoid lines like this.

Reply:
`git rebase --whitespace=fix` should fix it, too.

Reply:
Looks like Rider has a [nice feature for this](https://youtrack.jetbrains.com/issue/RIDER-9687/Add-option-to-trim-trailing-whitespace-on-save#focus=Comments-27-2437597.0-0). Thanks!